foundations/foundation-metrics/src/test/java/org/apache/servicecomb/foundation/metrics/publish/TestMetricsLoader.java

```diff
@@ -53,4 +54,15 @@ public void checkGetChildrenCount() {
     MetricNode node = loader.getMetricTree("X", "K1");
     Assert.assertEquals(2, node.getChildrenCount());
   }
+
+  @Test
+  public void checkNoSuchId() throws Exception {
+    try {
+      loader.getMetricTree("Z");
+      throw new Exception("CheckFailed");
+    }
+    //ignore because throw exception is correct
+    catch (ServiceCombException ignore) {
+    }
```

Comment:
We need to verify the ServiceCombException message here.

Reply:
Done